pkg/operation/botanist/component/etcd/bootstrap_test.go

```diff
@@ -161,17 +161,37 @@ rules:
   resources:
   - leases
   verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
   - create
+  -delete
```

Comment:
How can such test pass? Did you run `make verify` locally?

Reply:
Thanks for pointing out. My bad. it was failing in local as well